src/checkout/CancelButton.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import {CTAButton, ComponentColor} from '@influxdata/clockface'
+
+type ButtonClickEvent = React.MouseEvent<HTMLButtonElement>
+
+interface Props {
+  text?: string
+  onClick?: (e: ButtonClickEvent) => void
+}
+
+const noop = _ => {}
+
+const CancelButton: React.FC<Props> = ({onClick = noop, text = 'Cancel'}) => {
+  const handleClick = e => {
+    if (!!window?._abcr) {
+      window?._abcr.triggerAbandonedCart()
```

Comment:
This asset currently doesn't exist on the UI but we will need to integrate cart abandonment as part of this process

Reply:
Issue to integrate this here:

https://github.com/influxdata/ui/issues/877